cdap-api/src/main/java/co/cask/cdap/api/workflow/AbstractWorkflowAction.java

```diff
@@ -21,7 +22,7 @@
 public abstract class AbstractWorkflowAction implements WorkflowAction {
 
   private final String name;
-  private WorkflowContext context;
+  private WorkflowContext workflowContext;
```

Comment:
No need to rename, since we are already in the WorkflowAction, context would be of WorkflowContext.